src/Azure/Azure.Quantum.Client/SubmitterFactory.cs

```diff
@@ -138,14 +156,8 @@ public static class SubmitterFactory
         /// <param name="submitters">Information about each submitter.</param>
         /// <param name="target">The name of the execution target.</param>
         /// <returns>A name and type tuple.</returns>
-        private static (string Name, Type Type)? ConstructorInfo(IEnumerable<SubmitterInfo> submitters, string target)
+        private static (string Name, Type Type)? ConstructorInfo(SubmitterInfo submitter, string target)
```

Comment:
The `target` parameter is unused now. Also, this method no longer needs to return a nullable type since it never fails.

Reply:
Thanks for catching this! Fixing.